src/styles/_variables.scss

```diff
@@ -1,10 +1,12 @@
 $sp-1: 0.063rem;
```

Comment:
what's the origin for these variables?

Reply:
I have just converted pixels in rem vaia. say,  $sp-3: 0.188rem means 3px space converted in equivalent rem.